include/dem/particle_particle_contact_force.h

```diff
@@ -709,7 +710,7 @@ class ParticleParticleContactForce
     double coulomb_threshold =
       this->effective_coefficient_of_friction[vec_particle_type_index(
         particle_one_type, particle_two_type)] *
-      normal_force.norm();
+      std::fabs(normal_force_norm);
```

Comment:
same here. I don`t see how normal_force_norm can be negative.

Reply:
This is not a real norm